src/engine/strat_engine/filesystem.rs

```diff
@@ -23,6 +26,7 @@ pub struct StratFilesystem {
 
 pub enum FilesystemStatus {
     Good,
+    Warning,
```

Comment:
Ok.  I return the Warning in two cases.

1. the thin_dev.extend() failed.
2. xfs_growsfs() failed

In both cases we are NearCapacity, but your comment is making me think I should take it another step and have two states XfsGrowFailed, and ThinDevExpandFailed.  I think we are going to want to be specific with the dbus signals we issue.  

You agree?